new-tab-page/new-tab-page-api/src/main/java/com/duckduckgo/newtabpage/api/NewTabPagePlugin.kt

```diff
@@ -31,11 +31,14 @@ interface NewTabPagePlugin : ActivePlugin {
      * This method returns a [View] that will be used as the NewTabPage content
      * @param context The context to create the view with
      * @param showLogo Whether to show the logo in the new tab page
+     * @param onHasContent Optional callback to notify when the view has content
      * @return [View]
      */
-    fun getView(context: Context, showLogo: Boolean = true): View {
-        return getView(context)
-    }
+    fun getView(
+        context: Context,
+        showLogo: Boolean = true,
+        onHasContent: ((Boolean) -> Unit)? = null,
+    ): View
```

Comment:
Not sure why it’s pulling in these changes since they are already on develop, may need a rebase.